pkgs/top-level/all-packages.nix

```diff
@@ -3332,6 +3332,8 @@ with pkgs;
 
   bashblog = callPackage ../tools/text/bashblog { };
 
+  bergamot-models = recurseIntoAttrs (callPackages ../misc/bergamot-models { });
```

Comment:
(I'm not sure of the convention here.
Should it be `bergamot-models` or `bergamotModels`?)